docs/developer-guide/syntax-reference.rst

```diff
@@ -793,6 +793,8 @@ The PRINT statement supports the following properties:
 +-------------------------+------------------------------------------------------------------------------------------------------------------+
 | INTERVAL                | Print every nth message. The default is 1, meaning that every message is printed.                                |
 +-------------------------+------------------------------------------------------------------------------------------------------------------+
+| LIMIT                   | Stop printing after n messages. The default is to never stop printing and requires Ctrl+C to terminate.          |
++-------------------------+------------------------------------------------------------------------------------------------------------------+
```

Comment:
Should you add the the `INTERVAL interval` and `LIMIT limit` here too? Btw, LIMIT also supports the LIMIT ALL, so it would be good the explanation of ALL here too.

Reply:
Not sure why I can't reply to your comment below, but see discussion above. Decided to keep LIMIT ALL because it is supported in the normal query syntax. Whether or not we should have it should apply to both IMO :)